new_listings_scraper.py

```diff
@@ -66,11 +67,13 @@ def get_last_coin():
 
     uppers = None
 
-    if 'Will List' not in latest_announcement or found_coin[0] == existing_coin:
+    if 'Will List' not in latest_announcement or found_coin[0] == existing_coin or \
+            found_coin[0] in previously_found_coins:
         return None
     else:
         if len(found_coin) == 1:
             uppers = found_coin[0]
+            previously_found_coins.add(uppers)
```

Comment:
`previously_found` is a new set to stop a possible race condition of finding two annoucements close together, correct?